Sources/Helpers/OAAppSettings.m

```diff
@@ -1300,9 +1300,9 @@ - (instancetype) init
         [_autoZoomMap setModeDefaultValue:@NO mode:[OAApplicationMode PEDESTRIAN]];
         
         _mapSettingSaveTrackInterval = [OAProfileInteger withKey:mapSettingSaveTrackIntervalKey defValue:SAVE_TRACK_INTERVAL_DEFAULT];
-        [_mapSettingSaveTrackInterval setModeDefaultValue:@3000 mode:[OAApplicationMode CAR]];
-        [_mapSettingSaveTrackInterval setModeDefaultValue:@5000 mode:[OAApplicationMode BICYCLE]];
-        [_mapSettingSaveTrackInterval setModeDefaultValue:@10000 mode:[OAApplicationMode PEDESTRIAN]];
+        [_mapSettingSaveTrackInterval setModeDefaultValue:@3 mode:[OAApplicationMode CAR]];
```

Comment:
Is it related to RTL? Why SAVE_TRACK_INTERVAL_DEFAULT left 5000? What will be with old users and wrong values?

Reply:
Not related with RTL. Was noticed during RTL fixing, that default value in cell is set to "50 minutes". Tried to investigate. I think, problem is happened when figures were copied from Android, where time is measured in milliseconds.
SAVE_TRACK_INTERVAL_DEFAULT left 5000, because I didnt know about its existence. I think, it also should be changed to 5. 
What will be with old users and wrong values? - Cant answer this question.